Tests/StencilSwiftKitTests/ParseBoolTests.swift

```diff
@@ -8,17 +8,20 @@ import XCTest
 @testable import StencilSwiftKit
 
 class ParseBoolTests: XCTestCase {
-
   func testParseBool_TrueWithString() throws {
+    // swiftlint:disable force_unwrapping
     XCTAssertTrue(try Filters.parseBool(from: ["true"])!)
     XCTAssertTrue(try Filters.parseBool(from: ["yes"])!)
     XCTAssertTrue(try Filters.parseBool(from: ["1"])!)
+    // swiftlint:enable force_unwrapping
```

Comment:
Maybe we should instead use `XCTAssertTrue((try …) ?? false)` or `XCTAssertEquals(try …, true)` to avoid the force-unwrap here? Especially, in case there's a regression one day and the test  `parseBool(from: ["true"])` starts to fail and return an unexpected `nil`, that force-unwrap will make the test harness crash and stop the whole test process, instead of marking that test as failed and continue with other test cases…

Reply:
We'd have to be careful then to provide the "inverse" value as a fallback for each test. Here it's true, for the `parseBool(from: ["false"])` it should be false, ...

Reply:
Yeah that's why I'd prefer to use the more readable `XCTAssertEquals(…, true)` there rather than `XCTAssertTrue(… ?? false)`